src/Table/generate_property_tree/generate_property_tree.cxx

```diff
@@ -83,16 +87,29 @@ boost::property_tree::ptree tablator::Table::generate_property_tree(
 
     boost::property_tree::ptree &table_tree = resource.add(table_literal, "");
 
+    // Combine RESOURCE.TABLE.DESCRIPTION and comments in single string.
+    std::string description;
     for (auto &p : properties) {
         if (boost::starts_with(p.first, resource_literal + "." + table_literal)) {
-            for (auto &a : p.second.attributes)
+            for (auto &a : p.second.attributes) {
                 table_tree.add("<xmlattr>." + a.first, a.second);
+            }
+            if (boost::starts_with(p.first, resource_literal + "." + table_literal +
+                                                    "." + description_literal)) {
+                description.append(p.second.value);
+            }
         }
     }
     // VOTable only allows a single DESCRIPTION element, so we have to
     // cram all of the comments into a single line
```

Comment:
The comments are stuffed into a single DESCRIPTION element, not a single line - fix the comment while you are here?